src/datepicker/datepicker-navigation-select.ts

```diff
@@ -8,10 +8,8 @@ import {NgbDatepickerI18n} from './datepicker-i18n';
   changeDetection: ChangeDetectionStrategy.OnPush,
   styles: [`
     :host>select {
-      display: flex;
```

Comment:
it makes no sense to have `display: flex` on `select`. I tested this change in Google Chrome, Firefox and IE10, the selects look the same